_posts/2015-09-01-getting-started.html

```diff
@@ -314,6 +308,10 @@ <h6>Response</h6>
                         <td><strong>access_token</strong></td>
                         <td>Access Token (40 character length, hexa encoded sha1 hash)</td>
                     </tr>
+                    <tr>
+                        <td><strong>refresh_token</strong></td>
+                        <td>Refresh Token (40 character length, hexa encoded sha1 hash)</td>
```

Comment:
I don't think we should specify how we generate these tokens, something like "40 character long string" would be enough, but sadly this is not the only problem of this documentation :)